src/types/bearing_nb.cpp

```diff
@@ -0,0 +1,24 @@
+#include "engine/bearing.hpp"
+
+#include <nanobind/nanobind.h>
+#include <nanobind/operators.h>
+#include <nanobind/stl/pair.h>
+
+namespace nb = nanobind;
+
+void init_Bearing(nb::module_& m) {
+    using osrm::engine::Bearing;
+
+    nb::class_<Bearing>(m, "Bearing")
+        .def(nb::init<>())
+        .def("__init__", [](Bearing* t, std::pair<short, short> pair) {
```

Comment:
to me it's not immediately obvious what a bearing pair is. I guess we'll do docstrings once the API stabilized, still curious:)

also:
- can bearing be negative? or should it be `unsigned short`?
- can we use the more relatable aliases `uint16_t` etc?

Reply:
Bearing should be in [0; 360] range AFAIR.